### PR TITLE
Translate tsconfig-reference files to ko

### PR DESCRIPTION
**Pull request by bumkeyy**
Hi :)
It's first commit to translate typescript file to Korean!

This current PR translate the following files:
- [intro.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/tsconfig-reference/copy/en/intro.md)
- [files.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/tsconfig-reference/copy/en/options/files.md)

Ref : #910

### CODE REVIEW

**Comment by orta** [comment]
Hi - thanks!

So, I'd like to have someone else review this PR to make sure that the Korean makes sense. I only speak English (and Portuguese badly) so: @dvlprsh @yeonjuan @guyeol - you are top contributors to the Korean translation repo  

Could I maybe ask you to review this PR for how it sounds in Korean?

If you'd like to get involved more, we have a Korean translation channel in the [TypeScript Community Discord](https://discord.gg/typescript). Come say hi!

File: packages/tsconfig-reference/copy/ko/intro.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## TSConfig Reference 소개
+
+디렉터리의 TSConfig 파일은 현재 디렉터리가 TypeScript 또는 JavaScript 프로젝트의 루트 디렉터리임을 나타냅니다.
```
Comment (yeonjuan) [inline packages/tsconfig-reference/copy/ko/intro.md]:
```suggestion
디렉터리의 TSConfig 파일은 그 디렉터리가 TypeScript 또는 JavaScript 프로젝트의 루트 디렉터리임을 나타냅니다.
```

File: packages/tsconfig-reference/copy/ko/options/files.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+display: "Files"
+oneline: "Include a set list of files, does not support globs"
+---
+
+프로그램에 포함할 파일 허용 목록을 지정합니다. 파일을 찾지 못할 경우 오류가 발생합니다.
```
Comment (yeonjuan) [inline packages/tsconfig-reference/copy/ko/options/files.md]:
```suggestion
프로그램에 포함할 파일 허용 목록을 지정합니다. 파일을 찾지 못하면 오류가 발생합니다.
```

File: packages/tsconfig-reference/copy/ko/options/files.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "parser.ts",
+    "utilities.ts",
+    "binder.ts",
+    "checker.ts",
+    "tsc.ts"
+  ]
+}
+```
+
+파일의 수가 많지 않아 파일 참조에 glob을 사용할 필요가 없을 때 유용합니다. 필요한 경우 [`include`](#include)을 사용합니다.
```
Comment (yeonjuan) [inline packages/tsconfig-reference/copy/ko/options/files.md]:
```suggestion
파일 수가 적어 파일 참조에 glob을 사용할 필요가 없을 때 유용합니다. 필요한 경우 [`include`](#include)을 사용합니다.
```

**Review by yeonjuan — COMMENTED** [review]
@Bumkeyy 일부 표현이 부자연스러운 것 같아 suggestion 남겼습니다 :)

**Comment by bumkeyy** [comment]
@yeonjuan 
Thanks 👍

File: packages/tsconfig-reference/copy/ko/intro.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+## TSConfig Reference 소개
+
+디렉터리의 TSConfig 파일은 그 디렉터리가 TypeScript 또는 JavaScript 프로젝트의 루트 디렉터리임을 나타냅니다.
+TSConfig 파일은 `tsconfig.json` 또는 `jsconfig.json` 일 수 있으며, 둘 다 같은 동작과 설정 변수의 집합을 가지고 있습니다.
+
+이 페이지에서는 TSConfig 파일 내에서 사용할 수 있는 다양한 플래그를 모두 다룹니다. 모든 플래그의 개요를 시작으로 JSON 파일의 루트 속성을 지나, 옵션의 대부분인 `compilerOptions` 을 거쳐 `watchOptions`으로 마무리합니다.
```
Comment (dvlprsh) [inline packages/tsconfig-reference/copy/ko/intro.md]: @Bumkeyy 이렇게 수정하면 어떨까요..? 제안입니다 :)
```suggestion
이 페이지에서는 TSConfig 파일 내에서 사용할 수 있는 다양한 플래그를 모두 다룹니다. 모든 플래그의 개요를 시작으로 JSON 파일의 루트 속성을 살펴본 다음, 옵션의 대부분을 구성하는 `compilerOptions`를 거쳐 `watchOptions`로 마무리합니다.
```

**Review by yeonjuan — APPROVED** [review]
👍 LGTM

File: packages/tsconfig-reference/copy/ko/intro.md (hunk 4 above)
Comment (bumkeyy) [reply packages/tsconfig-reference/copy/ko/intro.md]: 훨씬 낫네요 👍

**Review by dvlprsh — APPROVED** [review]
LGTM 😄

**Comment by orta** [comment]
Thanks folks!

**Comment by orta** [comment]
This is live on staging: https://www.staging-typescript.org/ko/tsconfig

**Pull request merged**